### PR TITLE
1.0/336 endless spinner

### PR DESCRIPTION
**Pull request by gingerbreadpup**
Shortened the error messages for now - since the copy is just temporary we might as well have something that fits (in case of any more demos!). Once we decide on the final copy, I'll address the issue of making sure there's enough space to display the messages (already part of another card in tech debt).

Added fix for the spinner button so if you try to submit with no input, you get an error message and the button goes red and resets back to purple.

### CODE REVIEW

File: src/pages/Tap.vue [hunk 1]
```diff
@@ -77,11 +77,12 @@ export default {
...
                         ) {
                             this.showFail();
-                            this.errorMessage = "[xXx]The voucher code you entered is not valid. Please try again.";
+                            this.errorMessage = "[xXx] Please enter a valid voucher code.";
                         } else if (
                             response.data.fail.length > 0
                         ) {
-                            this.errorMessage = "[xXx]The voucher code you entered has previously been submitted. Please try again.";
+                            this.showFail();
+                            this.errorMessage = "[xXx] Please enter an unused voucher code.";
```
Comment (charlesstrange2) [inline src/pages/Tap.vue]: Makes me feel, as a user, like i didi this *on purpose* and am a bad person.
"That voucher may have been used already."

File: tests/pages/tap.test.js [hunk 2]
```diff
@@ -153,7 +153,7 @@ test('Correct error appears when I submit an invalid voucher', async t => {
         .click(submitVoucher)
     ;
     const errorMessage = await el('.message').innerText;
-    expect(errorMessage).to.contain('The voucher code you entered has previously been submitted. Please try again.');
+    expect(errorMessage).to.contain('Please enter an unused voucher code.');
```
Comment (charlesstrange2) [inline tests/pages/tap.test.js]: Change to suit the other changes above.

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
Code fine, not sure about the wording. A bit accusatory of the poor befuddled user people.

File: src/pages/Tap.vue (hunk 1 above)
Comment (gingerbreadpup) [reply src/pages/Tap.vue]: Sure, we can put whatever you like - mainly just wanted it to be shorter until we have approved copy. Will make changes now.

**Review by charlesstrange2 — APPROVED** [review]
:sheep::it:

**Pull request merged**